requirements.txt

```diff
@@ -14,7 +14,7 @@ Faker==0.8.11
 future==0.16.0
 hca==6.4.0
 git+git://github.com/hannes-ucsc/jmespath.py@hannes-fix-cache-race#egg=jmespath
-git+git://github.com/HumanCellAtlas/metadata-api@release/1.0b22#egg=hca-metadata-api
+git+git://github.com/DataBiosphere/hca-metadata-api@release/1.0b22#egg=hca-metadata-api
```

Comment:
For now this should point to you PR branch in that repo so we can see all the changes together. Once we merge that PR and release hca-metadata-api, this link should be adjusted again.